### PR TITLE
Rendertron

### LINKED ISSUE #45: [Feature] Serverside rendering of meta tags
**Is your feature request related to a problem? Please describe.**
Currently our project is rendered fully clientside. This is a problem, as for example twitterbot does not execute js and thus a hydration of the meta tags with dynamic content is not possible.

**Describe the solution you'd like**
I would like to have serverside rendering of meta tags

**Describe alternatives you've considered**
Sticking with the static tags implemented in https://github.com/TUM-Dev/navigatum/pull/43

### PR DESCRIPTION
**Pull request by CommanderStorm**
This PR adds [Rendertron](https://hub.docker.com/r/dockette/rendertron) a headless chrome proxy and screenshoting enginge
meta tags are added in #13/ https://github.com/TUM-Dev/navigatum/pull/43
resolves #45

### CODE REVIEW

**Review by octycs — APPROVED** [review]
So far, I cannot see any issues with this, thank you! Would you wait until #43 is merged before merging this?

**Comment by CommanderStorm** [comment]
We can do that

**Pull request merged**
